library/alloc/src/rc.rs

```diff
@@ -639,6 +655,225 @@ impl<T> Rc<T> {
     pub fn pin(value: T) -> Pin<Rc<T>> {
         unsafe { Pin::new_unchecked(Rc::new(value)) }
     }
+}
+
+impl<T, A: Allocator> Rc<T, A> {
```

Comment:
```suggestion
impl<T, A: Allocator> Rc<T, A> {
    /// Returns a reference to the underlying allocator.
    #[unstable(feature = "allocator_api", issue = "32838")]
    #[inline]
    pub fn allocator(&self) -> &A {
        &self.alloc
    }
```

like in https://doc.rust-lang.org/std/vec/struct.Vec.html#method.allocator

Reply:
I feel like this should have been more like https://doc.rust-lang.org/std/boxed/struct.Box.html#method.allocator than Vec's.

Reply:
You're 100% correct. Why did you say this 1 hour after it merged 😆 

Do you want to make a PR to fix this? If not, I can do one in a bit

Reply:
I say this 1 hour you merged it because I saw your message saying it merged.

There you go: https://github.com/rust-lang/rust/pull/113810

Reply:
Great, thanks!